### PR TITLE
[Term Entry] PHP Math Functions base_convert()

### PR DESCRIPTION
**Pull request by EspressoCat**
### Description

Added base_convert() term for PHP
Closes issue #2503 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@EspressoCat I did a first review and only had one comment to clarify the syntax.

**Comment by EspressoCat** [comment]
hey @codecademy-docs 👋🏻 

Let me know if requested changes are ok. Thanks!

File: content/php/concepts/math-functions/terms/base-convert/base-convert.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  - 'Math'
+  - 'Encoding'
+  - 'Numbers'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`base_convert()`** converts a number from one number base to another.
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/base-convert/base-convert.md:17]:
```suggestion
The **`base_convert()`** function converts a number from one base system to another.
```

File: content/php/concepts/math-functions/terms/base-convert/base-convert.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## Syntax
+
+```pseudo
+base_convert(string $number, int $frombase, int $tobase): string
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/base-convert/base-convert.md:22]:
```suggestion
base_convert($number, $fromBase, $toBase)
```

File: content/php/concepts/math-functions/terms/base-convert/base-convert.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+```
+
+The `base_convert()` function has three required parameters:
+
+- `$number`: This parameter specifies the number that is to be converted as a string.
+- `$frombase`: Specifies the original base of number.
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/base-convert/base-convert.md:28]:
```suggestion
- `$fromBase`: Specifies the original base of `$number` as an integer.
```

File: content/php/concepts/math-functions/terms/base-convert/base-convert.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+- `$tobase`: Specifies the base to convert to.
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/base-convert/base-convert.md:29]:
```suggestion
- `$tobase`: Specifies the base to convert to as an integer.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@EspressoCat I've added a few comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by EspressoCat** [comment]
Thanks for the comments @caupolicandiaz . 

Made requested changes to the entry.

**Comment by SSwiniarski** [comment]
@EspressoCat Congrats on the new entry! Live link here:https://www.codecademy.com/resources/docs/php/math-functions/base-convert

image

**Pull request merged**